test/rulesets/LinearAlgebra/symmetric.jl

```diff
@@ -305,7 +305,9 @@
             U = Matrix(qr(randn(T, n, n)).Q)
             if hermout # f(A) will also be a TA
                 λ = if f in (acos, asin, atanh)
-                    2 .* rand(real(T), n) .- 1
+                    # generate random number between -1 and 0.9. This would be between
+                    # -1 and 1 but we want to avoid domain errors from numerical error
+                    1.9 .* rand(real(T), n) .- 1
```

Comment:
Needs a T:
```
julia> 1.9 .* rand(real(Float32), 3) .- 1
3-element Vector{Float64}:
 -0.5762628734111785
 -0.7353183567523957
  0.8970144212245941
```

Also, unrelated to this, a few lines down defines lambda twice:
```

                λ = randn(real(T), n)
                λ = if f === atanh
                    2 .* rand(real(T), n) .- 1
                else
                    randn(real(T), n)
                end
                ```

Reply:
```suggestion
                    T(1.9) .* rand(real(T), n) .- 1
```